staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -305,6 +292,22 @@ func (o *CreateSecretOptions) createSecret() (*corev1.Secret, error) {
 	return secret, nil
 }
 
+// NewSecretObj will create a new Secret Object given name, namespace and secretType
+func NewSecretObj(name, namespace string, secretType corev1.SecretType) *corev1.Secret {
```

Comment:
I don't think this function needs to be exported, as it will only be used by this part of the code (aka it can be started with lower case)